pages/blog.js

```diff
@@ -1,23 +1,30 @@
 import { getAllFilesFrontMatter } from '@/lib/mdx'
 import siteMetadata from '@/data/siteMetadata'
-import ListLayout from '@/layouts/ListLayout'
+import BlogLayout from '@/layouts/BlogLayout'
 import { PageSeo } from '@/components/SEO'
 
+export const POSTS_PER_PAGE = 3
```

Comment:
3 is probably too little? It's good for testing but otherwise 8 or 10 seems more reasonable

Reply:
How do you want to approach this? Should users modify the `blog.js` file directly or use the file `siteMetadata.json` to set the amount of posts per pages?

Reply:
Modifying `blog.js` is fine